### ISSUE TITLE
saverudata.info

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/tile-attribution#9]
### Is it maybe reported already?

- [X] I have searched the existing issues, it is not reported already

### Before you submit this form, please ensure the following criteria are met:

- [X] The website/app in question uses map tiles hosted by the OpenStreetMap foundation (https://tile.openstreetmap.org).
- [X] The website/app does not have proper attribution. (https://osmfoundation.org/wiki/Licence/Attribution_Guidelines)
- [X] The usage of the tiles is non-trivial. (Not a development site, not test code, not a personal project with negligible traffic, etc.)
- [X] You have contacted the website/app owner or representative using the Love Letter model/template (https://osmfoundation.org/wiki/Licence/Attribution_Reminder_Templates) more than a week ago and no proper attribution has been added.
- [X] Or you have made a good faith effort to find working contact information for the website/app owner or representative but were unable to find a means to contact them.

### Date and time of the message (has it happened more than a week ago?):

01-Aug-2022 03:59:33 UTC

### Where did you send it?

N/A

### Please paste the content of the Love Letter you sent to the map user here.

N/A, previous contact by LWG.

### The infringement is a:

- [X] website
- [ ] app or other

### Where it is happening?

saverudata.info

**Comment by pnorman** [comment]
The above is an existing block from before this repo that just needs adding to the list here.

**Comment by Firefishy** [comment]
Duplicate of https://github.com/openstreetmap/tile-attribution/issues/3

**Comment by pnorman** [comment]
I've removed the block because it's not in the list and #3 was closed without adding them.

**Comment by matkoniecz** [comment]
https://github.com/openstreetmap/tile-attribution/issues/13#issuecomment-1842622175 implies that it was done on LWG request in 2022.

**Comment by deevroman** [comment]
Note that this domain originally hosted another site with leaked personal data.

But because it is an illegal site, it has many mirrors that still have access to the tiles:
https://saverudata.online/map/
https://saverudata.net/map/

**Comment by matkoniecz** [comment]
@deevroman Thanks for explanation and mentioning this mirrors, I contacted LWG about it.

**Comment by matkoniecz** [comment]
@deevroman In case you have not seen it: this mirrors were promptly blocked. If you see mirrors of this site or similar, feel free to report them.

**Comment by deevroman** [comment]
http://www.eduardbit.ru/map/
https://data.intelx.io/saverudata/map/

I think we can stop there - these seem to be the remaining active public mirrors.

**Comment by matkoniecz** [comment]
blocks issues, if they are not blocked within 24h please ping me here (I hope I configured blocks effectively this time)

**Comment by fabi003** [comment]
https://saverudata.online/map/ works for me with Firefox 121.0.1:

![Screenshot_20240118_175437](https://github.com/openstreetmap/tile-attribution/assets/110132764/45fbcb80-e426-480b-96a7-744644fba89b)

**Comment by Firefishy** [comment]
> https://saverudata.online/map/ works for me with Firefox 121.0.1:
>

Blocking isn't 100%. You likely have a privacy setting or privacy plugin which allows the tiles to work.
Most users will not see tiles.

**Comment by fabi003** [comment]
Yes, I use Referer-spoofing of Firefox and RFP mode beside from some content blocking and cookie isolation.

**Comment by Jean-BaptisteC** [comment]
It's seems website is down
https://saverudata.info/
